<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animate</class>
 <widget class="QDialog" name="Animate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/Icon16.png</normaloff>:/images/Icon16.png</iconset>
  </property>
  <widget class="QGraphicsView" name="AnimateView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string>Direction</string>
   </property>
   <widget class="QRadioButton" name="directLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="directRight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Animation</string>
   </property>
   <widget class="QRadioButton" name="Idle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Jump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump/float</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Sliding">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sliding</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Fire">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SitDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sit down</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dig</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RacoonRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Racoon run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RacoonFloat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Racoon float</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RacoonFly">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Racoon fly</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RacoonTail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Racoon tail</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="GrabRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="GrabJump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab jump</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="GrabSit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab sit down</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Swin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Swim</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Spin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Spin jump</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Climbing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Climbing</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TanookiStatue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanooki statue</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SwordAttak">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sword attak</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="JumpSwordUp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump - sword up</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="DownSword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Down+Sword attak</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="JumpSwordDown">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump - sword down</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Hurted">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hurted</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RideOnYoshi">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ride On Yoshi</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RideOnYoshiSit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ride On Yoshi (sit)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RideOnYoshiSit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pipe up/down</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SlopeSlide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Slope-Slide</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="EditAnimationBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit animations</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>directLeft</tabstop>
  <tabstop>directRight</tabstop>
  <tabstop>EditAnimationBtn</tabstop>
  <tabstop>Idle</tabstop>
  <tabstop>Run</tabstop>
  <tabstop>Jump</tabstop>
  <tabstop>Spin</tabstop>
  <tabstop>Sliding</tabstop>
  <tabstop>Climbing</tabstop>
  <tabstop>Fire</tabstop>
  <tabstop>SitDown</tabstop>
  <tabstop>Dig</tabstop>
  <tabstop>GrabRun</tabstop>
  <tabstop>GrabJump</tabstop>
  <tabstop>GrabSit</tabstop>
  <tabstop>RacoonRun</tabstop>
  <tabstop>RacoonFloat</tabstop>
  <tabstop>RacoonFly</tabstop>
  <tabstop>RacoonTail</tabstop>
  <tabstop>Swin</tabstop>
  <tabstop>TanookiStatue</tabstop>
  <tabstop>SwordAttak</tabstop>
  <tabstop>JumpSwordUp</tabstop>
  <tabstop>JumpSwordDown</tabstop>
  <tabstop>DownSword</tabstop>
  <tabstop>Hurted</tabstop>
  <tabstop>AnimateView</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
